go.mod

```
@@ -1,8 +1,9 @@
 module github.com/percona/percona-toolkit
 
-go 1.17
+go 1.18
```

I think we can use 1.19
```suggestion
go 1.19
```